src/goto-programs/safety_checker.h

```diff
@@ -45,4 +45,59 @@ class safety_checkert:public messaget
   const namespacet &ns;
 };
 
+/// \brief The best of two results
+inline safety_checkert::resultt operator||(
+  safety_checkert::resultt const &r1,
+  safety_checkert::resultt const &r2)
+{
+  return r1 == safety_checkert::resultt::SAFE
```

Comment:
How about defining an `operator<` then simply returning `std::min` or `std::max(r1, r2)`?

Reply:
(or a named comparator function, say `result_worse_than`, then using `std::min(r1, r2, result_worse_than)`

Reply:
Just realised this is only showing up at all due to clang-format, so can ignore this